.github/workflows/ci.yml

```diff
@@ -105,8 +105,29 @@ jobs:
             - name: Run tests
               shell: bash -l {0}  # required to activate the conda environment
               env: ${{ fromJSON(needs.build.outputs.images) }}
-              run: |
-                  pytest -v
+              run: pytest -v -m "not integration"
+
+    integration-test:
+        needs: build
+        runs-on: ubuntu-latest
+        timeout-minutes: 10
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Login to GitHub Container Registry
+              uses: docker/login-action@v2
+              with:
+                  registry: ghcr.io
+                  username: ${{ github.actor }}
+                  password: ${{ secrets.GITHUB_TOKEN }}
+            - name: Setup Python test environment
+              uses: mamba-org/provision-with-micromamba@v13
+              with:
+                  cache-env: true
+            - name: Run integration tests
+              shell: bash -l {0}  # required to activate the conda environment
+              env: ${{ fromJSON(needs.build.outputs.images) }}
+              run: pytest -v -m "integration"
```

Comment:
The name "integration" is a bit confuse because "I" for CI test is the meaning of integration. But I have no better suggestion TBH.

Reply:
I think the new tests fit the definition of 'integration tests' which is a standard term I think.

https://circleci.com/blog/unit-testing-vs-integration-testing/